docs/UserGuide.md

```diff
@@ -341,7 +342,6 @@ Exports a CSV file of the current view containing specified fields.
 Notes:
 * Name is always the first column in the CSV file.
 * The default exported fields are name, phone, email.
-* Invalid prefixes are ignored.
```

Comment:
Is this part onwards a duplicate of the other other PR #182 ?

Reply:
Oh sorry my bad!

Reply:
Ehh sorry my bad!

Reply:
J make sure if it merges it won't affect the other PR bah